Dockerfile

```diff
@@ -0,0 +1,132 @@
+# to build
+# docker build -t centos7_sal --rm=true .
+
+# Clean up
+# docker rm $(docker ps -a -q)
+
+
+FROM centos:7
+
+LABEL name="CentOS Base Image" \
+    vendor="CentOS" \
+    license="GPLv2" \
+    build-date="20161214"
+
+# Definitions
+ARG FITSIO_VERSION=1.0.4
+ARG SAL_VERSION=3.10.0_001
+ARG OSPL_VERSION=6.9.0
+ARG GIT_LSST="https://github.com/lsst-dm"
+ARG INSTALL_PATH=/opt/lsst
+ARG HEADERSERVICE_VERSION=0.9.7
+ARG SALPYTOOLS_VERSION=0.9.7
+ARG HSUSER=felipe
+
+RUN yum -y install https://centos7.iuscommunity.org/ius-release.rpm
+RUN yum -y install python36u python36u-devel python36u-pip
+RUN yum -y install python2-pip
+RUN yum -y install patch
+RUN yum -y install deltarpm
+RUN yum -y install xterm boost-python boost-python-devel maven  python-devel java-1.7.0-openjdk-devel
+RUN yum -y install epel-release
+RUN yum -y install python-astropy ipython
+RUN yum -y install wget
+RUN yum -y install git
+RUN yum -y install make
+RUN yum -y install emacs
+RUN yum -y install gcc-c++ gcc.x86_64 gcc-gfortran
+
+# Pip2/3 -- once we upgrade it's renamed pip3.6 --> pip3
+RUN pip install --upgrade pip
+RUN pip3.6 install --upgrade pip
+RUN pip3 install astropy
+RUN pip3 install ipython
+RUN pip3 install pyyaml
+RUN pip3 install fitsio==$FITSIO_VERSION
+
+# Make python3 link
+RUN ln -s /usr/bin/python3.6 /usr/bin/python3
+
+# ---------------------
+# ts_sal rpms
+# Add the lsst-ts repo
+RUN wget https://lsst-web.ncsa.illinois.edu/~felipe/packages/lsst-ts.repo
```

Comment:
This should be retrieved from another location, other than your own home directory. I suggest somewhere on GitHub, or in the lsst nexus repo.